README.md

```diff
@@ -25,6 +25,18 @@ Configuration
 
 Nothing special needed.
 
+Releases
+---
+
+Steps to build a new release:
+- Checkout the latest commit from the `8.x-1.x` branch.
+- Create a new branch, e.g., `release-8.1.6`.
+- Update the `composer.json` file to use the latest tagged release of all `su-sws` packages, eliminating the `@dev` dependencies.
```

Comment:
can we make this automatic. with github actions or something.

Reply:
+1 to automating the switch over on release branches but that can be a future improvement.